### ISSUE TITLE
UX: In SSKR printouts, mark words that can be left off of archives

### ISSUE THREAD

**Issue by ChristopherA** [BlockchainCommons/GordianSeedTool-iOS#163]
<img width="1050" alt="image" src="https://user-images.githubusercontent.com/69103/161849039-799c4803-2e66-4496-94c9-d5bec147853d.png">

It would be useful to mark or italicize the initial words "tuna acid epic gyro" as these are the same for all SSKRs. This means that these can be left off of metal plates that archive the remaining 25 words. This will work with most existing seed metal archive tools (all those that have room for the 25th BIP-38 passphrase.)

**Comment by wolfmcnally** [comment]
Please see my comment on this closely-related subject [here](https://github.com/BlockchainCommons/seedtool.info/issues/9#issuecomment-1112542744).

My strongly-held opinion is that the short-term convenience of having fewer words to inscribe into metal is far-outweighed by the robustness you are giving up by making the word reductions you propose:

* The structure is no longer valid CBOR.
* The structure is no longer self-identifying. With the tag words at the start, a future person will be able to discover their meaning by simply identifying the sequence of the first three words, which don't occur in that order anywhere in the world for any other purpose. Without them, they're just random words.
* The structure no longer contains error detection, nor can you rely as strongly on the last four words without the checksum words to reliably identify a particular share, as those last four words are purely the production of the CRC-32 algorithm, not the Shamir Secret Sharing algorithm.
* To reconstitute the structure users will have to add back the words, including the checksum, for all recovered shares.

In summary, attempting to recover such raw shares there is much more that can go wrong. All this risk to save a little engraving time and effort seems self-defeating when the stated goal is to accomplish robust backup and social recovery of highly valuable cryptographic data.
